### PR TITLE
Release V6.7.0 #8945

### LINKED ISSUE #8945: Release V6.7.0
Re-Scheduled for Wednesday, July 11, 2018, 10.00am SGT.

Sorry for the delay :)

### PR DESCRIPTION
**Pull request by xpdavid**
Part of #8945

Can't find the real name of some users in Github profile. Use their usernames as their real names.

### CODE REVIEW

File: src/main/webapp/data/developers.json [hunk 1]
```diff
@@ -1556,6 +1559,14 @@
       "multiple": true,
       "name": "Zhu Liang",
       "username": "paradite"
+    },
+    {
+      "name": "ah91086",
+      "username": "ah91086"
+    },
+    {
+      "name": "cybersol795",
```
Comment (wkurniawan07) [inline src/main/webapp/data/developers.json]: If the actual name cannot be found, leave the `name` field empty.

Comment (xpdavid) [reply src/main/webapp/data/developers.json]: Got it :)

File: src/main/webapp/data/developers.json [hunk 2]
```diff
@@ -312,6 +317,10 @@
       "name": "Connor McDonald",
       "username": "clmcdonald"
     },
+    {
+      "name": "",
```
Comment (wkurniawan07) [inline src/main/webapp/data/developers.json]: I mean, just omit the `name` field entirely. You can look at a few examples in this JSON file.

**Comment by xpdavid** [comment]
@damithc Can this one be merged?

**Pull request merged**